WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -170,7 +170,6 @@ class AztecPostViewController: UIViewController, PostEditor {
     ///
     fileprivate lazy var separatorButtonItem: UIBarButtonItem = {
         let separator = UIBarButtonItem(barButtonSystemItem: .fixedSpace, target: nil, action: nil)
-        separator.width = Constants.separatorButtonWidth
```

Comment:
How come we don't need to set a separator width any more?

Reply:
There is a default value, before we where hacking with a negative value to get some magic margins. That is not even officially allowed and I don't think we need to do it anymore